database_wrapper.py

```diff
@@ -408,14 +408,36 @@ def get_all_qa_pairs(self):
 
         return result
 
+    def get_all_unvalidated_qa_data(self, numQueries: int):
```

Comment:
Consider taking out the `_all` in the function name, since the number depends on the `numQueries` argument.

Reply:
Yup good catch